travis/build-and-test.sh

```diff
@@ -27,10 +27,19 @@ OLD_CWD=$(readlink -f .)
 pushd "$BUILD_DIR"
 
 # configure build
-cmake "$REPO_ROOT"
+if [ "MEASURE_CODE_COVERAGE" == "" ]; then
```

Comment:
This will never work. Also, that name is way too long. And it's generally not very elegant code. Check out the linuxdeploy build script.

Reply:
fixed